src/gui/MainWindow.cpp

```diff
@@ -70,7 +70,7 @@
 void disableAutoKeyAccelerators(QWidget* mainWindow)
 {
 	using DisablerFunc = void(*)(QWidget*);
-	QLibrary kf5WidgetsAddon("libKF5WidgetsAddons.so");
+	QLibrary kf5WidgetsAddon("KF5WidgetsAddons");
```

Comment:
Good, but it may not work on Ubuntu if `libkf5widgetsaddons-dev` is not installed. A simple solution is here:
```
// try with version number(for some systems like Ubuntu)
if (!kf5WidgetsAddon.load())
{
	kf5WidgetsAddon.setFileNameAndVersion("KF5WidgetsAddons", 5);
	kf5WidgetsAddon.load();
}
```

Reply:
@BumblingCoder May I push a commit addressing this? Or you may do that yourself if you want.

Reply:
If you want to pull this tonight you can do it, otherwise I can deal with it tomorrow evening. I think it is safe to always try to load version 5 though, so we could just throw the 5 in the constructor.

Reply:
> I think it is safe to always try to load version 5 though, so we could just throw the 5 in the constructor.

I'm not quite sure, but it should work if `libKF5WidgetsAddons.so.5` exists. Since the bug is a KDE 5 + Qt 5 issue, that looks safe for me. I'll wait for you, since you said you can do that tomorrow.